src/components/textField/index.js

```diff
@@ -800,7 +800,7 @@ function createStyles({centered, multiline}, rightItemTopPadding = 0) {
     },
     errorMessage: {
       color: Colors.red30,
-      textAlign: centered ? 'center' : undefined
+      textAlign: centered ? 'center' : 'left'
```

Comment:
We don't need a similar treatment here?

Reply:
Not from what I've tested, seems like `Text` (and probably `RNText`) is working properly unlike `RNTextInput`.